editor-components/stats/stats7/stats7.tsx

```diff
@@ -2,103 +2,119 @@ import * as React from "react";
 import { BaseStats } from "../../EditorComponent";
 import styles from "./stats7.module.scss";
 
-type ICard = {
-  subtitle: string;
+
+interface Item {
   title: string;
-  description: string;
-};
+  progress: number;
+  progressText: string;
+}
 
 class Stats7Page extends BaseStats {
   constructor(props?: any) {
     super(props, styles);
 
+    this.addProp({
+      type: "string",
+      key: "subtitle",
+      displayer: "subtitle",
+      value: "Our Skills."
+    })
+
     this.addProp({
       type: "string",
       key: "title",
       displayer: "Title",
-      value: "User Statistics on the Website",
+      value: "We work with organisations immersive customer.",
     });
     this.addProp({
       type: "string",
       key: "description",
       displayer: "Description",
       value:
-        "Understanding user behavior on a website is important for improving user experience and engagement.",
-    });
-    this.addProp({
-      type: "color",
-      key: "backgroundColor",
-      displayer: "Background Color",
-      value:
-        "#CDC1A9",
+        "Fierent abhorreant intellegam nam no. Eam minim di neglegentur te, ei etiamas corpora eam disentiun sea. Ut aeterno invidunt sententiae vel, assum adipisci eu vix. Ea ferri cetero ceteros eos, mea ne cibo dis entiet."
     });
     this.addProp({
       type: "array",
-      key: "card-content",
-      displayer: "Card Content",
+      key: "items",
+      displayer: "Items",
       value: [
         {
           type: "object",
-          key: "card",
-          displayer: "Card",
+          key: "item",
+          displayer: "Item",
           value: [
             {
               type: "string",
-              key: "subtitle",
-              displayer: "Subtitle",
-              value: "Products",
+              key: "progess-title",
```

Comment:
use camelCase for all keys. also keys must be same as you declared in type that you have written. ex:
```js
type Item {
 title: JSX.Element;
 progress: number;
 progressText: JSX.Element;
}
```